templates/data/opensearch.html

```diff
@@ -14,35 +14,36 @@
 
 <section class="p-strip is-shallow u-no-padding--bottom">
   <div class="p-section">
-  <div class="row--25-75">
-    <div class="col">
-      <div class="p-image-wrapper">
-        {{ image (
-          url="https://assets.ubuntu.com/v1/858e64ad-logo-open-search.svg",
-          alt="Opensearch",
-          width="40",
-          height="40",
-          hi_def=True,
-          loading="auto"
-          ) | safe
-        }}
-      </div>
-    </div>
-    <div class="col">
-      <div class="p-section--shallow">
-        <h1>OpenSearch<sup>&reg;</sup> operations, simplified</h1>
-        <p>Secure and automate the deployment, maintenance and upgrades of your search and analytics suite across private and public clouds.</p>
+    <div class="row--25-75">
+      <div class="col">
+        <div class="p-image-wrapper">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/858e64ad-logo-open-search.svg",
+            alt="Opensearch",
+            width="40",
+            height="40",
+            hi_def=True,
+            loading="auto"
+            ) | safe
+          }}
+        </div>
       </div>
-      <hr>
+      <div class="col">
+        <div class="p-section--shallow">
+          <h1>OpenSearch<sup>&reg;</sup> operations, simplified</h1>
+          <p>Secure and automate the deployment, maintenance and upgrades of your search and analytics suite across private and public clouds.</p>
+        </div>
+        <hr />
         <a class="p-button--positive" href="/contact-us" aria-controls="data-spark-modal">Contact us</a>
+        <a href="https://assets.ubuntu.com/v1/a7d1c19a-Charmed%20OpenSearch%20-%20Datasheet%20(1).pdf">Download the datasheet&nbsp;&rsaquo;</a>
```

Comment:
It would be better SEO and a11y to add some context to the link. Such as, "Download the Charmed OpenStack datasheet"

Reply:
But the text of the link says `Download the datasheet` already, shouldn't that be enough?

Reply:
If you feel the context of the link is clear enough?

Reply:
I think it is clear enough imo. The page is about Opensearch and the text of the link is quite self-explanatory.